lib/improver/blending/weighted_blend.py

```diff
@@ -846,8 +809,12 @@ def process(self, cube, weights=None,
                 blended with equal weights across the blending dimension.
             cycletime (str):
                 The cycletime in a YYYYMMDDTHHMMZ format e.g. 20171122T0100Z.
+                This can be used to manually set the forecast reference time
+                on the output blended cube. If not set, the most recent
+                forecast reference time from the contributing cubes is used.
             attributes_dict (dict or None):
-                Changes to cube attributes to be applied after blending
+                Changes to cube attributes to be applied after blending. See
+                improver.metadata.amend.amend_attributes for required format.
```

Comment:
It would be nice to make this a hyperlink, but even after asking Aaron, I'm not sure how.

Reply:
@gavinevans pointed me to the solution for this.

Reply:
@MoseleyS Sorry about that!
Glad it was figured out though - thanks @gavinevans